### DISCUSSION TITLE
how set-script

### DISCUSSION THREAD

**Discussion by l1mindr** [pnpm/pnpm#5683] [Q&A] [answered]
I want to know what is the equivalent of this command in pnpm?
```
npm set-script prepare ' '
```

**Comment by lvqq** [comment, marked as answer]
It's not supported currently, I have submitted a PR to add it: https://github.com/pnpm/pnpm/pull/5687

**Reply by lvqq** [reply]
It's released. Just update to pnpm v7.17.1

**Reply by l1mindr** [reply]
thank you
